### PR TITLE
Support key or range (MongoDB-style, IDBKeyRange) on server.count();

### PR DESCRIPTION
**Pull request by brettz9**
Add server-count tests and docs;
Utilize unminified version in test for easier debugging

I can revert the testing back to using the minified version of db.js if you like, but for testing, I think it makes sense to use the unminified.

### CODE REVIEW

**Comment by brettz9** [comment]
Note that I could also add key/range support to `query().count(key)`, but since there is already `range()` and the range methods, I wasn't sure you'd want the range support. I could add both or only the key support to `query().count(key)` if you like.

File: tests/views/layout.jade [hunk 1]
```diff
@@ -15,6 +15,6 @@ html
         script(src='../bower/jasmine/lib/jasmine-core/jasmine-html.js')
         script(src='../bower/jasmine/lib/jasmine-core/boot.js')
         script(src='../bower/jasmine-jsreporter/jasmine-jsreporter.js')
-        script(src='../dist/db.min.js')
+        script(src='../dist/db.js')
```
Comment (aaronpowell) [inline tests/views/layout.jade]: You should be getting source maps information so debugging won't be an issue. Are the source maps not working that you need to do this?

Comment (brettz9) [inline tests/views/layout.jade]: As I'm using Firefox, the console still doesn't utilize this, even though the Debugger does show the unminified source: https://bugzilla.mozilla.org/show_bug.cgi?id=670002

You want me to revert this?

Comment (aaronpowell) [inline tests/views/layout.jade]: Yep, I'd prefer tests to be against the minified release

**Comment by brettz9** [comment]
Ok, I've reverted back to db.min.js...

**Comment by brettz9** [comment]
Since I've reverted back to the minified file, can this be merged now?

**Comment by brettz9** [comment]
Thanks for the merge! Are you planning to merge your #151 soon? I'd like to prepare some new PR's, but don't know whether I should base it off of that or master...

**Pull request merged**